docs/vectorizer-quick-start-openai.md

```diff
@@ -0,0 +1,144 @@
+# Vectorizer quick start
+
+This page shows you how to create a vectorizer in a self-hosted Postgres instance, then use 
+the pgai vectorizer worker to create embeddings from data in your database. To finish off we show how simple it 
+is to do semantic search on the embedded data in one query!
+
+## Setup a local developer environment
+
+The local developer environment is a docker configuration you use to develop and test pgai, vectorizers and vectorizer
+worker locally. It includes a:
+- Postgres deployment image with the TimescaleDB and pgai extensions installed
+- pgai vectorizer worker image
+
+On your local machine:
+
+1. **Create the Docker configuration for a local developer environment**
+
+   Add the following docker configuration to `<timescale-folder>/docker-compose.yml`:
```

Comment:
IMO we should put this contents into a docker-compose file somewhere in the repo.